internal/api/v1/serviceclasses.go

```diff
@@ -0,0 +1,102 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/epinio/epinio/internal/cli/clients"
+	"github.com/epinio/epinio/internal/services"
+	"github.com/epinio/epinio/kubernetes"
+	"github.com/julienschmidt/httprouter"
+)
+
+type ServiceClassesController struct {
+}
+
+func (scc ServiceClassesController) Index(w http.ResponseWriter, r *http.Request) {
+	params := httprouter.ParamsFromContext(r.Context())
+	org := params.ByName("org")
+
+	cluster, err := kubernetes.GetCluster()
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+	gitea, err := clients.GetGiteaClient()
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+
+	exists, err := gitea.OrgExists(org)
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+	if !exists {
+		http.Error(w, fmt.Sprintf("Organization '%s' does not exist", org),
+			http.StatusNotFound)
+		return
+	}
+	serviceClasses, err := services.ListClasses(cluster)
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+
+	js, err := json.Marshal(serviceClasses)
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	_, err = w.Write(js)
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+}
+
+func (scc ServiceClassesController) ServicePlanList(w http.ResponseWriter, r *http.Request) {
```

Comment:
ServicePlans are another resources. It would be better if they had their own controller. Then the action would be:

```
func (sp ServicePlansController) List(...)
```

Reply:
Fixed it.